src/index.ts

```diff
@@ -85,6 +85,19 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     configurations.pollingInterval = MIN_POLLING_INTERVAL
   }
 
+  if (configurations.streamEnabled) {
+    try {
+      const { Platform } = require("react-native");
```

Comment:
Is there any issue with this require in a non react-native environment?

Reply:
tested in Chrome so far and testing in various other envs as I type this

Reply:
also tested on iOS/FF and Safari no problem there